controllers/example_controller/keyboard_controller.py

```diff
@@ -71,10 +71,13 @@ def print_sensors(robot: Robot) -> None:
                 print(
                     f" Spherical: {angle_str(rot_x)}, {angle_str(rot_y)}, {dist}",
                 )
-                rot_x, rot_y, rot_z = marker.orientation
+                roll = marker.orientation.roll
+                pitch = marker.orientation.pitch
+                yaw = marker.orientation.yaw
```

Comment:
Presumably this is expanded due to the bug in april_vision? Can we not just fix the bug?
https://github.com/WillB97/april_vision/issues/7 should be fixed by https://github.com/WillB97/april_vision/pull/9.

Reply:
No, this change was to make it clearer what the rotations are. Roll, pitch and yaw are understood by the layman and axis agnostic.